src/syft/core/common/module.py

```diff
@@ -77,3 +83,81 @@ def cpu(self) -> "Module":
         for _, module in self.modules.items():
             module.cpu()
         return self
+
+    # zero them so we know they are copied
+    def zero_layers(self) -> None:
+        for _, m in self.modules.items():
+            if hasattr(m, "weight"):
+                m.weight.requires_grad_(False).zero_()
+            if hasattr(m, "bias"):
+                m.bias.requires_grad_(False).zero_()
+
+    # easy way to check the weights have changed
+    def sum_layers(self) -> None:
```

Comment:
Q: If we use this for debugging maybe it would be better to add a proper name?
like ```def debug_sum_layers``` - is this meant to be used by a dev?

Reply:
Yes this is likely temporary, so renaming is a good idea.